javaLegacyApp/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">javalegacyapp</string>
```

Comment:
This will show in the Launcher, may want to make prettier

Reply:
it will be the correct name in FA-114/116 , right now its been deleted